guides/guide.txt

```diff
@@ -0,0 +1,18 @@
+TOPIC
+    guide
+
+DESCRIPTION
+    A guide is a person who leads travelers, sportsmen, or tourists through
+    unknown or unfamiliar locations. The term can also be applied to a person who
+    leads others to more abstract goals such as knowledge or wisdom. 
+
+    Etymology: Originated sometime between 1325 and 1375. From Middle English
+    guide, from the Old French guide, from Old Occitan guida, from guidar, from
+    Frankish *wītan (“to show the way, lead”), from Proto-Germanic *wītaną (“to
+    see, know; go, depart”), from Proto-Indo-European *weyd- (“to see, know”).
+    Cognate with Old English wītan (“to see, take heed to, watch after, guard,
+    keep”). Related also to English wit. 
```

Comment:
```suggestion
    Etymology: Originated sometime between 1325 and 1375. From Middle English
    guide, from the Old French guide, from Old Occitan guida, from guidar, from
    Frankish *wītan (“to show the way, lead”), from Proto-Germanic *wītaną (“to
    see, know; go, depart”), from Proto-Indo-European *weyd- (“to see, know”).
    Cognate with Old English wītan (“to see, take heed to, watch after, guard,
    keep”). Related also to English wit. 
```
I feel like we're going into the weeds a little here? It's whimsical, so if that's the intent, I'm fine with it, but I wonder if we should put a paragraph above this one that tells people what Bolt guides are for? 
Something like:
> Several of Bolt's commands offer guides that include basic usage information, command options, and links to documentation.

Reply:
Maybe I'm just missing the joke cos it's Monday morning and I shouldn't do PR reviews first thing on a Monday morning :D

Reply:
Hmm, the intent is definitely to be whimsical more than to be helpful - this guide won't even show in the `bolt guide` output that lists available guides, so it's intended to be an easter egg. Is there a way to make that more clear do you think? Maybe some ASCII art? Definitely still open to a short graf about bolt commands!

Reply:
Haha, I mean, there's not *that* much of a joke to get, this is just copied from the linked wikis. I think as an easter egg it could use some more creativity 😅

Reply:
Yeah, gotcha :D I figured it was an easter egg after I'd had some coffee. ASCII art would be amazing TBH!

Reply:
Hm, I noodled on this for a little bit and at this point am out of creativity - I think we should either cut this and add it later, or just merge as is :/ @beechtom thoughts on which?

Reply:
You can always merge it as is and then iterate on it for a hack day or something. This is the kind of thing I could hack at too :D

Reply:
Agree with Heston, I think we can merge this for now and iterate on it later, especially since it's just an easter egg. :)